advisor/src/main/kotlin/Advisor.kt

```diff
@@ -62,10 +65,10 @@ class Advisor(private val providerFactory: VulnerabilityProviderFactory, private
             "The provided ORT result file '${ortResultFile.canonicalPath}' does not contain an analyzer result."
         }
 
-        val provider = providerFactory.create(config)
+        val providers = providerFactories.map { it.create(config) }
 
         val packages = ortResult.getPackages(skipExcluded).map { it.pkg }
-        val results = runBlocking { provider.retrievePackageVulnerabilities(packages) }
+        val results = runBlocking { providers.first().retrievePackageVulnerabilities(packages) }
```

Comment:
To be frank, I'm not happy with the topical split between this and the next commit. That's mostly because you already expose the feature to use multiple advisors to the user here, but only make it work in the next commit.

It would have been somewhat fine to still prepare the implementation to be able to deal with multiple advisors here, not implement it fully, but then also not expose it to the user yet, and only expose it to the user in the next commit when it's really working.

But actually, as the next commit is not too big, I wonder why we can't also simply squash these two renaming commits.

Reply:
Yes, I agree that the topical spilt is not ideal. I have no problems with squashing, so I will do it.